app/generators/range_generator.rb

```diff
@@ -33,6 +33,18 @@ def range_slider(question)
                      max: minmax[:max].to_s,
                      step: question[:step] || 1,
                      required: true)
+    # TODO: Make this code prettier.
+    unless question[:value].nil?
+      range_body = tag(:input,
+                       type: 'range',
+                       id: idify(question[:id]),
+                       name: answer_name(idify(question[:id])),
+                       min: minmax[:min].to_s,
+                       max: minmax[:max].to_s,
+                       step: question[:step] || 1,
+                       required: true,
+                       value: question[:value])
+    end
```

Comment:
Kan je al die properties niet gewoon in een hash zetten, en daar dan evt. dat value attribuut aan toevoegen?
En dan roep je `tag` aan met `:input` en dan die hash.
Dat scheelt een boel codeduplicatie.

Reply:
```

  def range_slider(question)
    minmax = range_slider_minmax(question)
    question_options = {
      type: 'range',
      id: idify(question[:id]),
      name: answer_name(idify(question[:id])),
      min: minmax[:min].to_s,
      max: minmax[:max].to_s,
      step: question[:step] || 1,
      required: true
    }
    question_options[:value] = question[:value] if question[:value].present?
    range_body = tag(:input, question_options)
    tag.p(range_body,
          class: 'range-field')
  end
```